CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 FEATURES:
+* **New Data Source**: d/tfe_organization_workspace_tags is a new datasource to allow reading workspace tags within an organization, by @rhughes1 ([#773](https://github.com/hashicorp/terraform-provider-tfe/pull/773))
```

Comment:
Thanks for contributing this! I only see one adjustment I'd like you to make to be more in keeping with other resource conventions. Can you rename this to `tfe_organization_tags`?

Reason being, it isn't scoped to a particular workspace and better reflects usage of the actual endpoint, [Organization Tags](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organization-tags)

Eventually, we'll want to add arguments that reflect the other capabilities of the endpoint like tag name filtering, but that doesn't diminish the usefulness of the data source. Thanks again!

Reply:
Of course! The reason I called it `organization_workspace_tags` was _in case_ the new projects feature would eventually get the ability to tag we wouldn't have to rename this data source. I called it `organization_workspace_tags` so it was very explicit to end users.